bundles/org.openhab.binding.hapero/README.md

```diff
@@ -0,0 +1,146 @@
+# hapero Binding
+
+This Binding is intended for [Spanner Re²/Hapero](https://www.holz-kraft.com/) Central Heating Systems
+It collects data from the FTP upload feature of the furnace control system. (The "Upload.hld" file).
```

Comment:
`It collects data from the FTP upload feature of the furnace control system (the "Upload.hld" file).`

Reply:
OK, fixed!